packages/@aws-cdk/aws-codeguruprofiler/test/integ.profiler-group.ts

```diff
@@ -0,0 +1,30 @@
+import { AccountRootPrincipal, Role } from '@aws-cdk/aws-iam';
+import { App, Stack, StackProps } from '@aws-cdk/core';
+import { ProfilingGroup } from '../lib/profiling-group';
```

Comment:
This way you can verify that you indeed export the types:

```suggestion
import { ProfilingGroup } from '../lib';
```